### DISCUSSION TITLE
Failure to deposit the assembly

### DISCUSSION THREAD

**Discussion by hadiabniki** [iotaledger/firefly#4040] [Help & Support]
After updating the Firefly wallet, the assemblies acquired in the second stick have been removed

**Comment by andreser27** [comment]
The same happened to me. I reinstalled the previous version and the asmb appeared again. But if I install the latest version, the asmb will disappear

**Reply by HankMoody089** [reply]
This bug is known and will be fixed in an upcoming version.
